pkg/builder/builder.go

```diff
@@ -59,6 +59,8 @@ type Builder struct { // Do not add comments for this struct
 	APITestTemplates []*schemas.APITestTemplate
 }
 
+var armTypeList = []string{"a1", "m6g", "m6gd", "t4g", "c6g", "c6gd", "c6gn", "r6g", "r6gd", "x2gd"}
```

Comment:
This can be listed with API. 
`aws ec2 describe-instance-types --filters Name=processor-info.supported-architecture,Values=arm64` 

This would be better for next update.

You can find the go sdk [here](https://docs.aws.amazon.com/sdk-for-go/api/service/ec2/#EC2.DescribeInstanceTypes).

Reply:
```
 aws ec2 describe-instance-types --filters Name=processor-info.supported-architecture,Values=arm64 --query "InstanceTypes[*].InstanceType"
```

Reply:
Thank you for advice. I implement that using aws api, get arm64 instanceType and compare user input of override spot instance type